ui/tests/acceptance/enterprise-namespaces-test.js

```diff
@@ -9,21 +9,57 @@ import {
   visit,
   fillIn,
   currentURL,
-  waitFor,
   findAll,
   triggerKeyEvent,
   find,
 } from '@ember/test-helpers';
 import { module, test } from 'qunit';
 import { setupApplicationTest } from 'ember-qunit';
-import { runCmd, createNS } from 'vault/tests/helpers/commands';
+import { runCmd, createNS, deleteNS } from 'vault/tests/helpers/commands';
 import { login, loginNs, logout } from 'vault/tests/helpers/auth/auth-helpers';
 import { AUTH_FORM } from 'vault/tests/helpers/auth/auth-form-selectors';
 import { GENERAL } from '../helpers/general-selectors';
 import { NAMESPACE_PICKER_SELECTORS } from '../helpers/namespace-picker';
```

Comment:
Up to you, but I like to shorten then. For Secret Engine Selectors I used SES, or maybe here just NAMESPACE_PICKER. Shortening it saves the smallest amount of time when typing these out, those co-pilot kind of does it for you. Again, totally up to you.